flasksrc/__init__.py

```diff
@@ -102,14 +103,20 @@ def create_app(test_config=None):
                      '/api/toynet/session/<int:toynet_session_id>/delete/link')
     api.add_resource(ToyNetSessionByIdDeleteDevice,
                      '/api/toynet/session/<int:toynet_session_id>/delete/<string:device_type>')
+    api.add_resource(ToyNetSessionByIdCreateRouterInterface,
+                     '/api/toynet/session/<int:toynet_session_id>/create/router/interface')
+    api.add_resource(ToyNetSessionByIdDeleteRouterInterface,
+                     '/api/toynet/session/<int:toynet_session_id>/delete/router/interface')
     docs.register(ToyNetSession)
     docs.register(ToyNetSessionById)
     docs.register(ToyNetSessionByIdTerminate)
     docs.register(ToyNetSessionByIdCreateHost)
     docs.register(ToyNetSessionByIdCreateSwitch)
     docs.register(ToyNetSessionByIdCreateRouter)
+    docs.register(ToyNetSessionByIdCreateRouterInterface)
     docs.register(ToyNetSessionByIdCreateLink)
     docs.register(ToyNetSessionByIdDeleteLink)
     docs.register(ToyNetSessionByIdDeleteDevice)
+    docs.register(ToyNetSessionByIdDeleteRouterInterface)
```

Comment:
**Nitpick:** ordering here seems a bit strange. 

We could align by `<action, device>`:
* `CreateHost`
* `CreateSwitch`
* `CreateRouter`
* `CreateRouterInterface`
* `CreateLink`
* `DeleteDevice`
* `DeleteRouterInterface`
* `DeleteLink`

Or alphabetically:
* `CreateHost`
* `CreateLink`
* `CreateRouter`
* `CreateRouterInterface`
* `CreateSwitch`
* `DeleteDevice`
* `DeleteLink`
* `DeleteRouterInterface`

Reply:
Grouped by verb/noun and made the `api.add_resource` calls match